class.jetpack-cli.php

```diff
@@ -944,6 +946,10 @@ public function partner_provision( $args, $named_args ) {
 			$request_body['force_connect'] = intval( $named_args['force_connect'] );
 		}
 
+		if ( 0 < $request_body['onboarding'] ) {
```

Comment:
is `$request_body['onboarding']` going to be anything other than 1? Maybe to identify different hosts?